hack/allocate.sh

```diff
@@ -271,43 +274,13 @@ EOF
 
 }
 
-# temporary fix for https://github.com/dapr/cli/issues/1311
-fixup_dapr_dashboard_deployment() {
-  kubectl patch deploy/dapr-dashboard -n dapr-system --patch-file /dev/stdin <<EOF
-{
-  "spec": {
-    "template": {
-      "spec": {
-        "\$setElementOrder/containers": [
-          {
-            "name": "dapr-dashboard"
-          }
-        ],
-        "containers": [
-          {
-            "image": "ghcr.io/dapr/dashboard:0.12.0",
-            "name": "dapr-dashboard"
-          }
-        ]
-      }
-    }
-  }
-}
-EOF
-}
-
 dapr_runtime() {
   echo "${em}⑦ Dapr${me}"
-
-  local dapr_flags=""
-  if [ "${GITHUB_ACTIONS:-false}" = "true" ]; then
-    (sleep 10; fixup_dapr_dashboard_deployment)&
-    dapr_flags="--image-registry=ghcr.io/dapr --log-as-json"
```

Comment:
I think using the `ghcr.io` registry and `--log-as-json` is still useful.

Reply:
at least in GH Action

Reply:
Sure, that part is added back.  Just kept the removal of line 304; the invocation of the dapr dashboard patch.